src/css/transformHEXRGBa.ts

```diff
@@ -11,17 +11,62 @@ export function transformHEXRGBaForCSS(token: DesignToken): DesignToken['value']
   const type = token.$type ?? token.type;
   if (val === undefined) return undefined;
 
-  const transformHEXRGBa = (val: string) => {
-    const regex = /rgba\(\s*(?<hex>#.+?)\s*,\s*(?<alpha>\d*(\.\d*|%)*)\s*\)/g;
-    return val.replace(regex, (match, hex, alpha) => {
-      try {
-        const [r, g, b] = new Color(hex).srgb;
-        return `rgba(${r * 255}, ${g * 255}, ${b * 255}, ${alpha})`;
-      } catch (e) {
-        console.warn(`Tried parsing "${hex}" as a hex value, but failed.`);
-        return match;
+  const transformHexColor = (hex: string) => {
+    try {
+      // Fast path for invalid hex
+      if (hex.length < 4) return hex;
+
+      // Determine format based on length
+      const hexLength = hex.length - 1; // subtract 1 for #
```

Comment:
```suggestion
      // Determine format based on length
      const hexLength = hex.length - 1; // subtract 1 for #
      
      // Fast path for invalid hex
      if (hexLength < 3) return hex;
      
```
This makes more sense, otherwise as a reader of the code I'd be wondering "what about hex3, 3 is less than 4"